src/AddNode.scss

```diff
@@ -75,3 +75,7 @@ button.btn.AddNode__expanded-button svg {
     width: (2 * $addNodeDropMargin) + $widthOfCollapsedIcon;
     height: $addNodeDropAreaHeight;
 }
+
+.AddNode__expanded-button.isDragTarget {
```

Comment:
We're using BEM naming for our classes, please rename to `.AddNode__expanded-button--isDragTarget`

Reply:
Still working on this one, I think I'm missing another location where the old `isDragging` is added/removed.

Reply:
i.e. if I change the class in the two places I've found, the styles stop working.

Reply:
Maybe I can pair with @chosww briefly on this on Monday.

Reply:
This does still need addressing, AFAIK it's the only thing left.

Reply:
Figured it out.  The updated BEM selector wasn't specific enough to trump the styles imposed on button.btn.X, I had to add `!important` to the rule.